corehq/apps/sms/tests/test_backends.py

```diff
@@ -353,6 +354,7 @@ def test_outbound_sms(
         self._test_outbound_backend(self.karix_backend, 'karix test', karix_send)
         self._test_outbound_backend(self.airtel_tcl_backend, 'airtel tcl test', airtel_tcl_send)
 
+    @flaky
```

Comment:
Might be good to mark this entire class. I've also seen twilio fail

Reply:
573df880682

Reply:
...coincidentally, `test_push_inbound_sms` just failed on this PR.